epidose/device/wps_scanner_d.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+#
+# Check for available networks and connect to the closest one using the WPS PBS
+#
+# Copyright 2020 Stefanos Georgiou
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+export APP_NAME=wps_scanner_d
+
+# Location of suspend watchdod led
+SUSPEND_WATCHDOG_LED=/var/lock/epidose/suspend-watchdog-led
+
+# Location of update_filter_d sleep process file
+SLEEP_UPDATE_FILTER_PID=/var/run/epidose_update_filter_sleep
```

Comment:
Define it in` util.sh` and use it on both scripts, right?

Reply:
Done!